int8            inter_id        # Intersection ID
int8[]          robot_id_list   # List of robot names or IDs
RobotInfo[]     robot_info      # List of RobotInfo instances